fearless/Common/Helpers/AssetBalanceDisplayInfo.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import SoraFoundation
+
+struct AssetBalanceDisplayInfo {
+    let preferrableDisplayPrecision: UInt16
```

Comment:
Does "preferable" mean that we would like to see it that way, but it is not guaranteed? If not, I would rename it to "displayPrecision"